clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/model/Job.java

```diff
@@ -131,12 +133,14 @@ public Job(
     applicationName = grpcJob.getJobDescriptor().getContainer().getImage().getName();
     version = grpcJob.getJobDescriptor().getContainer().getImage().getTag();
     digest = grpcJob.getJobDescriptor().getContainer().getImage().getDigest();
-    entryPoint =
-        grpcJob.getJobDescriptor().getContainer().getEntryPointList().stream()
-            .collect(Collectors.joining(" "));
-    cmd =
-        grpcJob.getJobDescriptor().getContainer().getCommandList().stream()
-            .collect(Collectors.joining(" "));
+
+    entryPointArray =
```

Comment:
The `*Array` naming is throwing me a bit, Titus is calling it `entryPointList`?

Is there a reason not to have `entryPoint` and `entryPoints`? `cmd` and `cmds`?

Reply:
Yh. I struggled with that name. `entryPoints` or `entryPointList` sounded like there can be more than one entry point where as in the real world those are kind of parts of what makes an entryPoint and same is true for cmd. The final payload that gets sent to titus is a list/array of strings that defines the full entryPoint and cmd.
Docker refers to those as exec form array so I ended up using that
```The exec form is parsed as a JSON array, which means that you must use double-quotes (“) around words not single-quotes (‘).```

Reply:
@ajordens :  made it as `entryPointList` and `cmdList` similar to titus api.